data/sql_updates/create_sched_e_aggregate_candidate.sql

```diff
@@ -9,8 +9,10 @@ select
     sum(exp_amt) as total,
     count(exp_amt) as count
 from sched_e
-where exp_amt is not null
-and (memo_cd != 'X' or memo_cd is null)
+where
+    exp_amt is not null and
+    filing_form in ('F3X', 'F5') and
```

Comment:
Maybe the filtering is going on somewhere else, but we only want form_tp="SE" from the sched_e table